build.gradle.kts

```diff
@@ -73,6 +73,7 @@ dependencies {
     api("com.atlassian.performance.tools:jira-actions:[2.2.0,4.0.0)")
     api("com.github.stephenc.jcip:jcip-annotations:1.0-1")
 
+    implementation("com.atlassian.data:random-data:[1.1,2.0.0)")
```

Comment:
Does the lib obey semVer? I didn't find any API docs in the library's docs?

It's dangerous to use ranges for libraries that don't provide any API guarantees.

Reply:
At the moment virtual-users are the only user of this library, so I will keep them updated. But yeah, eventually, this library will be used in the data generator, so I will start semver-ing it.

Reply:
I'll just switch to a fixed version for now.